src/updater/__init__.py

```diff
@@ -2,69 +2,131 @@
 from dotenv import load_dotenv
 from datetime import datetime as dt
+import logging
 
 class Updater():
-    def __init__(self, county):
+    def __init__(self, county = "hays"):
         self.county = county.lower()
+        self.case_json_cleaned_folder_path = os.path.join(
+            os.path.dirname(__file__), "..", "..", "data", self.county, "case_json_cleaned"
+        )
+        self.processed_path = os.path.join(self.case_json_cleaned_folder_path)
+        
+        # open or create a output directory for a log and successfully processed data
+        if os.path.exists(self.case_json_cleaned_folder_path) and \
+            not os.path.exists(self.processed_path): 
+            os.makedirs(self.processed_path)
+        self.logger = self.configure_logger()
 
-    def update(self):
+        self.COSMOSDB_CONTAINER_CASES_CLEANED = self.get_database_container()
 
-        case_json_cleaned_folder_path = os.path.join(
-            os.path.dirname(__file__), "..", "..", "data", self.county, "case_json_cleaned"
+    def configure_logger(self):
+        logger = logging.getLogger(name="pid: " + str(os.getpid()))
+        logger.setLevel(logging.DEBUG)
+
+        cleaner_log_path = os.path.join(
+            os.path.dirname(__file__), "..", "..", "resources"
         )
 
-        list_case_json_files = os.listdir(case_json_cleaned_folder_path)
-        limiter = 0
-        #Loops through all of the cleaned and redacted JSON files (the final versions)
-        for case_json in list_case_json_files:
-            limiter +=1
-            if limiter == 5:
-                break
-            print(case_json)
-            # Opens the JSON file and reads it to a dictionary.
-            in_file = case_json_cleaned_folder_path + "\\" + case_json
-            with open(in_file, "r") as f:
-                input_dict = json.load(f)
-            print(input_dict)
-            #This loads the environment for interacting with CosmosDB #Dan: Should this be moved to the .env file?
-            load_dotenv()
-            URL = os.getenv("URL")
-            KEY = os.getenv("KEY")
-            DATA_BASE_NAME = os.getenv("DATA_BASE_NAME")
-            CONTAINER_NAME_CLEANED = os.getenv("CONTAINER_NAME_CLEANED")
+        file_handler = logging.FileHandler(os.path.join(cleaner_log_path, 'logger_log.txt'))
+        file_handler.setLevel(logging.DEBUG)
+        formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+        file_handler.setFormatter(formatter)
+        logger.addHandler(file_handler)
+
+        console_handler = logging.StreamHandler()
+        console_handler.setLevel(logging.WARNING)
+        console_handler.setFormatter(formatter)
+        logger.addHandler(console_handler)
+
+        return logger
+    
+    def get_database_container(self):
+        #This loads the environment for interacting with CosmosDB #Dan: Should this be moved to the .env file?
+        load_dotenv()
```

Comment:
Hm, just looking at the docs and it seems like you may want to use `dotenv_values(".env")` here instead of loading the env, just to make this a little more isolated.

https://pypi.org/project/python-dotenv/#:~:text=Other%20Use%20Cases-,Load%20configuration%20without%20altering%20the%20environment,-The%20function%20dotenv_values

So the following lines could read..

```python
env = dotenv_values(".env") # I assume `env` isn't a reserved word

URL = env["URL"]
URL = env("URL")
KEY = env("KEY")
DATA_BASE_NAME = env("DATA_BASE_NAME")
CONTAINER_NAME_CLEANED = env("CONTAINER_NAME_CLEANED")
```

Reply:
You could also just move `load_dotenv()` to the top of this module, I think.